Description: Fix endian build problem on ppc64el
  Drop endian override in powerpc architecture definition
  allowing both big and little endian builds.
Author: Andy Whitcroft <apw@ubuntu.com>
Origin: ubuntu
Forwarded: no
Reviewed-By: Petter Reinholdtsen <pere@hungry.com>
Last-Update: 2016-05-12

Index: zfs-linux-0.6.5.3/lib/libspl/include/sys/isa_defs.h
===================================================================
--- zfs-linux-0.6.5.3.orig/lib/libspl/include/sys/isa_defs.h	2015-10-27 14:36:37.904604893 -0600
+++ zfs-linux-0.6.5.3/lib/libspl/include/sys/isa_defs.h	2015-10-27 14:36:58.008605392 -0600
@@ -98,8 +98,8 @@
 #endif
 #endif
 
-#if !defined(_BIG_ENDIAN)
-#define	_BIG_ENDIAN
+#if !defined(_BIG_ENDIAN) && !defined(_LITTLE_ENDIAN)
+#error powerpc no endian specified
 #endif
 
 #define	_SUNOS_VTOC_16
